apps/src/templates/NavigationBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import NavigationBarLink from './NavigationBarLink';
+
+const styles = {
+  navBar: {
+    height: 50,
+    width: '100%',
+    backgroundColor: color.purple,
+  },
+  linkBox: {
+    display: 'inline',
+    paddingRight: 20,
+    paddingLeft: 20,
+  },
+};
+
+export default class NavigationBar extends React.Component {
+  static propTypes = {
+    links: PropTypes.array,
+    activeLink: PropTypes.string,
```

Comment:
If we're going to have `activeLink` as both a prop and state, maybe we should follow the [React uncontrolled components](https://reactjs.org/docs/uncontrolled-components.html) convention of naming the prop `defaultActiveLink` so it's obvious from outside that updating the prop does _not_ necessarily change the active link.